File: Uri Online Judge/C/1002.c
#include <stdio.h>

#define PI 3.14159

int main()
{
    float raio = 0,
        area = 0;

    scanf(" %f", &raio);
    area = PI * (raio * raio);

    printf("A=%.4f\n", area);

    return 0;
}

File: Uri Online Judge/Rust/1002.rs
fn main() {
    let pi = 3.14159;

    use std::io::{stdin};
    let mut entrada = String::new();
    
    stdin().read_line(&mut entrada).expect("");
    let raio: f64 = entrada.trim().parse().expect("Not numeric");

    let area = pi * raio * raio;

    println!("A={:.4}", area);
}
